ui/package.json

```diff
@@ -39,6 +39,7 @@
         "react-dom": "18.2.0",
         "react-router-dom": "6.10.0",
         "styled-components": "5.3.9",
+        "url": "0.11.3",
```

Comment:
Required by v6 AWS Amplify - Get an error from vite if this is not added as dependency